.hashibot.hcl

```diff
@@ -7,8 +7,36 @@ behavior "remove_labels_on_reply" "remove_stale" {
     labels = ["waiting-reply"]
 }
 
+poll "stale_issue_closer" "waiting_reply_closer" {
+    schedule = "1 0 * * *"
+    no_reply_in_last = "480h" # 20 days
+    max_issues = 50
+    sleep_between_issues = "5s"
+    labels = ["waiting-reply-🤖"]
+    message = <<-EOF
+    Hey there,
+
+    This issue has been automatically closed because there hasn't been any activity for at least _20_ days.
+
+    If you are still experiencing problems, or still have questions, feel free to [open a new one](https://github.com/hashicorp/consul/issues/new) :+1:.
+    EOF
+}
+
+poll "stale_issue_closer" "old_issue_closer" {
+    schedule = "*/5 * * * *"
```

Comment:
Same as comment below, the scheduling of hashibot needs an overhaul, to avoid rate limits I would prefer this be configured to run daily at a particular time say 4 o'clock (I think the heroku app is in UTC).
```
schedule = "0 0 16 * * *"
max_issues = 1000
```
You should be able to crank up the max issues per run to 1000.